server/server.js

```diff
@@ -33,6 +38,16 @@ app.use('/api/photos', photosRouter);
 app.use('/api/steps', stepsRouter);
 app.use('/api/teams', teamsRouter);
 app.use('/api/contest', contestRouter);
+app.use('/api/dailychallenge', dailyChallengeRouter);
+app.use('/api/challengephotos', challengePhotosRouter);
+
+// app.use('/', s3Router);
+app.use('/s3', UploaderS3Router({
```

Comment:
![image](https://user-images.githubusercontent.com/1153371/98959104-9235f180-24c8-11eb-8402-a130b3aadb70.png)
that's it, huh?